README.md

```diff
@@ -8,7 +8,7 @@
 [![Pod Platform](https://img.shields.io/cocoapods/p/AppAuth.svg?style=flat)](https://cocoapods.org/pods/AppAuth)
 [![Catalyst compatible](https://img.shields.io/badge/Catalyst-compatible-brightgreen.svg?style=flat)](https://developer.apple.com/documentation/xcode/creating_a_mac_version_of_your_ipad_app)
 
-AppAuth for iOS and macOS is a client SDK for communicating with 
+AppAuth for iOS and macOS, and tvOS is a client SDK for communicating with 
```

Comment:
```suggestion
AppAuth for iOS, macOS, and tvOS is a client SDK for communicating with 
```
Or we can simplify it to just "AppAuth for iOS", @WilliamDenniss thoughts?

Reply:
On the name, I think "AppAuth for iOS, macOS, and tvOS" is a bit of a mouthful, but does makes sense. Maybe this is the full name, and "AppAuth for iOS" is the short name?

Reply:
There's currently (limited) watchOS support as well...  "AppAuth for Apple"?